modules/backup-and-restore/pages/point-in-time-restore.adoc

```diff
@@ -0,0 +1,76 @@
+
+[console, gsql]
+---
+$ gadmin backup list
+|================================================================================================================================================
+|             TAG              |    TYPE     | VERSION |  SIZE  |             BASE             | POINT-IN-TIME START TIME |      CREATED AT      
+| backup-2024-06-13T061939.131 | FULL        | 4.1.0   | 1.5 MB |                              | 2024-06-13T06:19:39Z     | 2024-06-13T06:19:39Z 
+| backup-2024-06-13T063915.054 | INCREMENTAL | 4.1.0   | 1.5 MB | backup-2024-06-13T061939.131 | 2024-06-13T06:19:39Z     | 2024-06-13T06:39:15Z 
+| backup-2024-06-13T081939.120 | FULL        | 4.1.0   | 1.5 MB |                              | 2024-06-13T08:19:39Z     | 2024-06-13T08:19:39Z 
+|================================================================================================================================================
+---
+
+here, the time range for backup `backup-2024-06-13T061939.131` is `[2024-06-13T06:19:39Z,2024-06-13T06:19:39Z]`, therefore, for a full backup is just a point. As for the incremental backup `backup-2024-06-13T063915.054`, its range is `[2024-06-13T06:19:39Z , 2024-06-13T06:39:15Z]`. so, for instance, if we have performed a restore at `2024-06-14T06:19:39Z`. And then we can’t restore to the timepoint `2024-06-13T07:00:00Z`, for it is not in any backup's time range. however, `2024-06-13T06:35:00Z` is supported for it is covered by the time range `[2024-06-13T06:19:39Z , 2024-06-13T06:39:15Z]` of the backup `backup-2024-06-13T063915.054`.
+
+
+[NOTE]
+====
+because the restore action will make the uncovered timepoint unavailable to restore, if you are not very sure about the time point to restore,  we’d better ensure the possible time points are all covered by existing backups. if there is not this kind of backup, you may create a by `gadmin backup create <backup_tag> --incremental --base <full_backup_tag>`, the new backup will cover the range from the full backup <full_backup_tag>'s create time to the current time.
+====
+
+== Key Benefits
+
+The primary advantage of Point-in-Time Restore is the flexibility it offers in selecting any specific time point on the timeline for restoration.
+
+.Key Benefit Scenario
+====
+. A user creates a full backup on __Sunday__.
+
+. During the week, user creates a differential backup on every day 10pm.
+
+. On Thursday, user finds there are unwanted changes made on Wednesday around 2pm.
+
+. User can restore system to 1:55pm of Wednesday instead of the backup made on Tuesday 10pm.
+
+. The changes made between Tuesday 10pm and Wednesday 1:55pm are saved this way.
+====
+
+Summery of key benefits:
+
+* Precise recover time point to save a much as possible correct work made to database.
+
+
+== Limitations and Considerations
+
+Currently, the Point-in-Time Restore feature has these known limitations:
+
+.Point-in-Time Restore examples.
```

Comment:
For requests like RESTPP request or query, they will fail, as starting GPE/GSE is the last step of restore, TG cannot serve those requests during restore. User will know restoring finished through information printed out at console.

@runlilong Please help answer what will happen if run gadmin command while restoring. And any other ways to know restore is finished?

Reply:
> Please help answer what will happen if run gadmin command while restoring.

the consequence is dependent on the concrete command and the fire-out time. if the command has nothing to do with restore flow and its dependent services are online,  the command can respond normally, such as `gadmin status`. however, it would fail if the restore flow is exactly at the gap of stop Kafka.  and if the gadmin command is to stop ctrl, then the restore flow will be failed.  in short, we assume there wouldn't be significant gadmin command during the restore.

> And any other ways to know restore is finished?

the restore command completion means the restore is done.

Reply:
I won't add anything to the doc right now, but we should ask the TSE team what advice they would give to users and then add that.